app/models/account.rb

```diff
@@ -1,3 +1,4 @@
 class Account < ApplicationRecord
   belongs_to :project
+  belongs_to :account_manager, class_name: 'User'
 end
```

Comment:
Innen hiányzik egy csomó validáció.

#61 
ld.:
- `Service Name` (required)
- `Services` (required - would be nice to store it as jsonb list)
- `Who manages the account?` (required - must be one or more users associated with the project)
- `Do you need to own an account?` (required, boolean)

Good to know:
- `Project / Organization` a fenti `belongs_to` megköveteli a jelenlétét
- required boolean változót nem `presence` kulcsszóval validálunk
- Nem tudom, hogy szükséged lesz-e a custom validációra, de ezeket privát függvényként szoktuk definiálni.
- Itt érdemes keresgélned: https://guides.rubyonrails.org/active_record_validations.html